src/ia-menu-slider.js

```diff
@@ -87,20 +88,22 @@ export class IAMenuSlider extends LitElement {
   }
 
   get renderMenuHeader() {
-    const { title = '', menuDetails = '', actionButton } = this.selectedMenuDetails || {};
+    const { label = '', menuDetails = '', actionButton } = this.selectedMenuDetails || {};
     const actionSection = actionButton
       ? html`<div class="custom-action">${actionButton}</div>`
       : nothing;
 
     return html`
       <header>
         <div class="details">
-          <h3>${title}</h3>
+          <h3>${label}</h3>
           <span class="extra-details">${menuDetails}</span>
         </div>
         ${actionSection}
         <button class="close" aria-label="Close this menu" @click=${this.closeMenu}>
-          <ia-icon-collapse-sidebar></ia-icon-collapse-sidebar>
+          <ia-icon-collapse-sidebar
+            style="--iconWidth: var(--closeIconWidth); --iconHeight: var(--closeIconHeight);"
```

Comment:
These styles can be moved to src/styles/menu-slider.js, like:
```css
  ia-icon-collapse-sidebar {
    --iconWidth: var(--closeIconWidth);
    --iconHeight: var(--closeIconHeight);
  }
```

Reply:
done